hotel-management/.eslintrc.cjs

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  root: true,
+  env: { browser: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:import/errors',
+    'plugin:import/warnings',
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+  ],
+  ignorePatterns: ['dist', '.eslintrc.cjs'],
+  parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
+  settings: {
+    react: { version: '18.2' },
+    'import/resolver': {
+      typescript: {
+        project: './tsconfig.json',
+      },
+    },
+  },
+  plugins: ['react-refresh', '@typescript-eslint', 'import'],
+  rules: {
+    'react/jsx-no-target-blank': 'off',
```

Comment:
Why turn off this rule?

Reply:
It is off by default when I initialize the app, I was searching on Internet and this explain why: react/jsx-no-target-blank: off will allows dev to use target="_blank" without adding an extra attribute for security and Eslint won't warning about it

Reply:
Make no sense when default this rule was off